eecs338.sublime-project

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
that's for me, been too long since using VIM so nowadays I use sublime text with a project file in each repo